Jenkinsfile

```diff
@@ -0,0 +1,78 @@
+pipeline {
+  environment {
+    devRegistry = 'ghcr.io/datakaveri/rs-dev'
```

Comment:
Do not use the rs image name, change to lip. check already existing ghcr package name for lip